src/components/grid/gridRowContent.js

```diff
@@ -10,6 +10,16 @@ import { View, Text, Drawer, Touchable } from '@keg-hub/keg-components'
 import { useSessionLocation } from 'SVHooks/models'
 import { BookingButton } from 'SVComponents/button/bookingButton'
 import { SessionPresenters } from 'SVComponents/sessionDetails'
+import { getBookingState } from 'SVUtils/models/sessions/getBookingState'
+import { Values } from 'SVConstants/values'
+import { useStoreItems } from 'SVHooks/store/useStoreItems'
+
+const { SESSION_BOOKING_STATES } = Values
+
+const useSessionState = (session)=>{
+  const attendees = useStoreItems('attendees')
+  return useMemo(() => getBookingState(session), [session,attendees])
+}
```

Comment:
Hmm, our eslinter doesn't seem to be working on the commits. Can you add some spaces here? Sorry, I know it's very minor but consistency is always nice

```suggestion
const useSessionState = (session) => {
  const attendees = useStoreItems('attendees')
  return useMemo(() => getBookingState(session), [session, attendees])
}
```

Reply:
I'm pretty sure the linter doesn't run until it's merged.
Otherwise, any changes it made would have to be commited to the PR branch
Which means, the remote PR branch would now be 1 commit ahead of your local.
So you would have to remember to pull down the changes every time you made an update to the code locally.

That said, I'm pretty sure this spacing would be fixed by the linter, once the code is merged upstream

Reply:
Made changes anyway.Now in stateLabel file. However, Would you like me to leave this in without spaces and verify if it is formatted correctly when merged?